build/generate-manifests.sh

```diff
@@ -41,37 +41,57 @@ function checkDependencies() {
     echo "Please install yq package, e.g. 'pip install --user yq'"
     exit 1
   fi
-}
 
-function generateServiceMeshControlPlanesCrd() {
-  yq -s -y --indentless '.[] | select(.kind=="CustomResourceDefinition" and .metadata.name=="servicemeshcontrolplanes.maistra.io") | .' ${DEPLOYMENT_FILE} > ${BUNDLE_DIR}/servicemeshcontrolplanes.crd.yaml
+  if ! [ -x "$(command -v operator-sdk)" ]; then
```

Comment:
As we have a dependency on operator-sdk, could we just `go run ...`?

Reply:
I'm not sure how one can add a dependency to `operator-sdk/cmd`. We currently don't have that package in `vendor/`. Any ideas?

Reply:
Look at `tools.go`  I think you can add an import there for the tools.  I think this might be how the newer versions of operator-sdk setup the project anyway.

Reply:
Thanks! Importing it works, but until we upgrade the operator-sdk, we can't use it like this. So, I'm leaving things as they for now.

Reply:
I've now fixed this. `operator-sdk` is run via `go run`.